subsys/bluetooth/mesh/shell/shell.c

```diff
@@ -1299,10 +1399,7 @@ static int cmd_dst(const struct shell *sh, size_t argc, char *argv[])
 	int err = 0;
 
 	if (argc < 2) {
-		shell_print(sh, "Destination address: 0x%04x%s", bt_mesh_shell_target_ctx.dst,
-			    bt_mesh_shell_target_ctx.dst == bt_mesh_shell_target_ctx.local
-				    ? " (local)"
-				    : "");
+		shell_print(sh, "Destination address: 0x%04x", bt_mesh_shell_target_ctx.dst);
```

Comment:
Why was it changed?

Reply:
It is not valid if the internal prov context is not used. It could be conditional print for when CONFIG_BT_MESH_SHELL_PROV_CTX_INSTANCE is enabled, but I choose the simple fix here.

Reply:
I think by using `bt_mesh_primary_addr()` we can get rid of the `local` parameter all togheter.